### PR TITLE
Bugfix in DijSampling

### PR DESCRIPTION
**Pull request by eric11210**
Old behaviour: if all bixels were in the core, the outer-core sampling
would still be attempted, resulting in an error message when attempting
to do the linspace (since the radDepthTail would be empty).

New behaviour: if all bixels are in the core, exit function with core
dose only.  Otherwise, proceed to rest of function.

### CODE REVIEW

**Comment by markbangert** [comment]
I think this looks good. @becker89 could you also have a quick look? And @eric11210 yould you maybe remove your gitignore file from the commit? Then I could just merge directly online ;)

File: matRad_DijSampling.m [hunk 1]
```diff
@@ -80,47 +80,56 @@
...
-    ixSamp             = r<=(subDose/thresholdDose);
-    NumSamples         = sum(ixSamp);
-
-    ixNew(IxCnt:IxCnt+NumSamples-1,1)        = subIx(ixSamp);    % save new indices
-    bixelDoseNew(IxCnt:IxCnt+NumSamples-1,1) = thresholdDose;    % set the dose
-    IxCnt = IxCnt + NumSamples;    
+if all(ixCore)
+    %% all bixels are in the core
+    %exit function with core dose only
+    ixNew = ix(ixCore);
```
Comment (becker89) [inline matRad_DijSampling.m]: I think we could also simply say:   ixNew = ix;

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+*.fig
```
Comment (becker89) [inline .gitignore]: Could you exclude the gitignore file from the pull request. It's better in my opinion if users use their own customized gitignore file.

**Comment by markbangert** [comment]
Done.

**Comment by eric11210** [comment]
Sorry Mark, I've been busy the last couple of days. Do you still want me to resubmit a pull request without the gitignore?

Eric

**Comment by markbangert** [comment]
@eric11210 don't worry about it. I am just in the process of cleaning up a lot of matRad stuff so I will revise myself and perform the merge.

thanks again for submitting this bug!

**Pull request merged**